### PR TITLE
Add tests pertaining to tab switching feature

### LINKED ISSUE #285: Add test cases for tab switching feature

### LINKED ISSUE #343: Add tests for MainWindow

### PR DESCRIPTION
**Pull request by ngzhenteng**
Fixes #285

### CODE REVIEW

File: src/test/java/seedu/jarvis/logic/commands/AddConsultationCommandTest.java [hunk 1]
```diff
@@ -47,6 +47,18 @@ public void execute_duplicateConsultation_throwsCommandException() {
                 AddConsultationCommand.MESSAGE_DUPLICATE_CONSULTATION, () -> addCommand.execute(modelStub));
     }
 
+    @Test
```
Comment (peironggg) [inline src/test/java/seedu/jarvis/logic/commands/AddConsultationCommandTest.java:50]: Great usage of stubs

File: src/test/java/seedu/jarvis/logic/commands/CommandResultTest.java [hunk 2]
```diff
@@ -51,4 +51,47 @@ public void hashcode() {
         // different exit value -> returns different hashcode
         assertNotEquals(commandResult.hashCode(), new CommandResult("feedback", false, true).hashCode());
     }
+
```
Comment (peironggg) [inline src/test/java/seedu/jarvis/logic/commands/CommandResultTest.java:54]: Real comprehensive test suite!

File: src/test/java/seedu/jarvis/ui/MainWindowTest.java [hunk 3]
```diff
@@ -1,31 +1,62 @@
 package seedu.jarvis.ui;
```
Comment (peironggg) [inline src/test/java/seedu/jarvis/ui/MainWindowTest.java:1]: Thats alot of tests brother

Comment (ngzhenteng) [reply src/test/java/seedu/jarvis/ui/MainWindowTest.java:1]: haha lets hope we bump codecov up

Comment (wireseo) [reply src/test/java/seedu/jarvis/ui/MainWindowTest.java:1]: Yeah this is a lot HAHAHA

**Pull request merged**